_posts/2016-10-02-distributed-array-of-commodity-hosting-services.md

```diff
@@ -0,0 +1,184 @@
+---
+layout: post
+title:  "Distributed Array of Commodity Hosting Services (DACHS)"
+date:   2016-10-02 15:29:37
+categories: definition architecture web commodity
+---
+
+<span><q>[This is] not [suitable for] enterprise production use where an
+enterprise such as ourselves would want an SLA.</q> -- Anonymous Architect</span>
+{: class="look-at-me"}
+
+The above quote shocked me when I was describing to some people how this works.
+These ideas wouldn't seem to be as well-known as I thought, so this post is an
```

Comment:
Are "these ideas" here the ideas posited by the Google paper which is mentioned later? I have a little subject confusion.